_includes/email-subscribe-form.html

```diff
@@ -8,15 +8,32 @@
       enctype="application/x-www-form-urlencoded">
 
     <div class="form-group">
-        <label class="u-visually-hidden" for="email-subscribe-form_email">
+        <label for="email-subscribe-form_email">
             Email address
         </label>
         <input class="reveal-on-focus_target"
                id="email-subscribe-form_email"
                type="email"
                name="email"
-               placeholder="Your email">
+               placeholder="example@mail.com">
     </div>
+    {# Uncomment this and remove condition to test
```

Comment:
Is this still needed? What needs to be uncommented and what is being tested when doing so?